src/post_process/m_global_parameters.fpp

```diff
@@ -474,7 +474,7 @@ contains
         integer :: i, j, fac
 
         ! Setting m_root equal to m in the case of a 1D serial simulation
-        if (num_procs > 1 .and. n == 0) m_root = m_glb
+        if (n == 0) m_root = m_glb
```

Comment:
There must be some reason why that if statement was in there.... do you know what it was?

Reply:
There was a line in the `s_decompose_computational_domain.fpp` in `src/post_process/m_mpi_proxy.fpp` that set `m_root = m` _before_ `m` was converted from a global cell count to a local cell count (https://github.com/MFlowCode/MFC/blob/40164cf2e1a13d5f77a42ee50d39d7d2bdb24f2e/src/post_process/m_mpi_proxy.fpp#L573), so `m_root` was always being set to `m_glb`. In 1D, post processing is done by doing an MPI reduce to the root processor and saving only from the root.